base/linalg/generic.jl

```diff
@@ -491,18 +504,30 @@ function normInf{T}(A::AbstractMatrix{T})
 end
 
 """
-    norm(A, [p::Real=2])
+    norm(A::AbstractArray, p::Real=2)
+
+Compute the `p`-norm of a vector or the operator norm of a matrix `A`,
```

Comment:
On second thought "induced operator norm" is redundant, so kindly ignore this comment. Either "induced norm" or "operator norm" seems best per your point below :).